locales-pending/premium.ftl

```diff
@@ -362,6 +362,7 @@ premium-upsell-dialog-title = Choose the level of protection that’s right for
 guided-resolution-flow-exit = Return to dashboard
 guided-resolution-flow-back-arrow = Go to previous step
 guided-resolution-flow-next-arrow = Go to next step
+guided-resoultion-flow-step-navigation-label = Guided steps
```

Comment:
Not sure if this label makes sense to users (nor localisers)? Maybe something more like "Data to remove" or something? Or ask Mel :sweat_smile: 

(Hmm, I'm now also wondering if `<nav>` is even the right element, since you can't actually use it to go elsewhere, but that's out of scope of this PR.)

Reply:
I was wondering about that as well. We also have the step indicators in the onboarding wrapped in a `<nav>` element. Just a list should do — I’ll follow-up.

Reply:
Btw, there was also [this interesting thread](https://mozilla.slack.com/archives/C4E0W8B8E/p1697228044912049?thread_ts=1697131641.050789&cid=C4E0W8B8E) recently about who should write accessible text labels. And of course, the ideal would be for Mel to do that. I've been kinda avoiding going back to ask about this and just came up with labels myself, since I don't know what experience they have, but it might be worth giving a shot?

Reply:
Usually, I also don’t go back to UX and content to ask about labels. I like the idea of thinking about the labels more as part of a feature that we include in the initial spec! Making them up on the go can lead to only describing the functionality and not the context they are in by using terminology we might only use within code.